include/nbl/ext/ScreenShot/ScreenShot.h

```diff
@@ -59,19 +93,15 @@ inline core::smart_refctd_ptr<asset::ICPUImageView> createScreenShot(video::ILog
 	info.commandBuffers = &gpuCommandBuffer.get();
 	info.pSignalSemaphores = nullptr;
 	info.signalSemaphoreCount = 0u;
-	info.pWaitSemaphores = &semaphore;
-	info.waitSemaphoreCount = 1u;
+	info.pWaitSemaphores = nullptr;
+	info.waitSemaphoreCount = 0u;
```

Comment:
because `pWaitSemaphores` waits (in the driver or GPU onboard command processor) for a semaphore BEFORE the submit (latches the download of the image) and the FENCE waits on the CPU for the image download commandbuffer to finish executing

Assign to @Erfan-Ahmadi